docs/UserGuide.md

```diff
@@ -511,22 +511,32 @@ Ryan; Phone: 91234567; Email: ryan@gmail.com; Address: My Secret Home; Selling A
 
 ### Sorting buyers: `bsort`
 
-Sorts the buyers' list by the provided attribute and in the given direction.
-
-Format: `bsort ATTRIBUTE_PREFIX DIRECTION`
-* `ATTRIBUTE_PREFIX`: Refer to the add buyer command, `buyer`
+Sorts the buyers' list either by the list's default order, or by a provided prefix and direction.
+
+Format: `bsort [PREFIX/DIRECTION]`
+* `PREFIX`: Choose <u>one</u> of:
```

Comment:
No need for this, i've made an appendix to explain the fields. Just link it here in an info box.

Reply:
I think it is good to have it here for the user to refer to it more easily though!